### ISSUE TITLE
[Feature]: Adding a top x search list to front page

### ISSUE THREAD

**Issue by narbs91** [narbs91/glosseta#32]
### Context

A feature request came in for a list of the top searched terms of a given day to be displayed on the front page to give users a potential starting point for their curiosity. 

### Problem and/or Opportunity

To be able to better serve peoples curiosities (i.e. those people who don't know where to start) a feature was proposed to add a top x search term list to the front page so people had a place to start going down the rabbit hole.  The list should ideally be created from the most searched terms of a given day but can be made a rotating list of terms for the time being.

### Proposed Solution

We will look to add a small list of terms that a user can click into that would sit on the landing page but just below the search bar.  For an initial implementation we can look to make it a rotating list of terms given some parameter (maybe time of day) and have each term be a link into its definiton.  Ideally we'd want this feature to be driven by the search terms people are actually looking up on a given day so some effort can be spent to see the different ways that can be achieved.

### Testing the changes

Testing the changes will involve adding unit tests, smoke testing the new changes to test it's functionality and making sure that existing functionality isn't broken

**Comment by G3root** [comment]
Is there any way to fuzzy search arweave graphql API ?

**Comment by narbs91** [comment]
I don't think there is a way just re-reading the [documentation](https://gql-guide.vercel.app/).  The way the document suggests searching is to use it to search for transactionId's and then taking that transactionId and querying arweave directly.  I went a slightly different route with my design where i stuck basically everything as a tag (thus metadata) and preform the search that way.  The nature of the data is such that I can get away with storing everything I need as a tag; however, it's not ideal (or at least the way i would do it in web2) since basically the definition is not only the data but part of the metadata as well.  I did that so i can reduce the number of network calls i needed to make so i can live with the trade off.
